CHANGELOG.unreleased.md

```diff
@@ -8,12 +8,11 @@ and this project adheres to [Calendar Versioning](http://calver.org/) `0Y.0M.MIC
 For upgrade instructions, please check the [migration guide](MIGRATIONS.released.md).
 
 ## Unreleased
-[Commits](https://github.com/scalableminds/webknossos/compare/22.11.1...HEAD)
+[Commits](https://github.com/scalableminds/webknossos/compare/22.11.2...HEAD)
 
 ### Added
 
 ### Changed
-- When merging annotations, bounding boxes are no longer duplicated. [#6576](https://github.com/scalableminds/webknossos/pull/6576)
```

Comment:
I branched of `release-22.11.1` which is not identical to the tag 22.11.1 (since the master was pulled into the branch before merging into the master). This is why this change is now also contained in 22.11.2. I don't think, it's a problem which is why I kept it. Feel free to veto.